python/ccxt/base/decimal_to_precision.py

```diff
@@ -169,6 +169,8 @@ def number_to_string(x):
     # avoids scientific notation for too large and too small numbers
     if x is None:
         return None
+    if isinstance(x, str):
```

Comment:
@ttodua I don't think we can do this because we do some transformations to strings here
![image](https://github.com/ccxt/ccxt/assets/43336371/f53903ac-1aab-4afc-890e-12628daf9ed5)

Reply:
but in other langs we return the input if it's not `numeric` type. is that expected in other langs?

Reply:
hum, I don't think so they should behave similarly across the different languages